eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Page.php

```diff
@@ -75,25 +75,6 @@ public function preMap(array $config, ContextualizerInterface $contextualizer)
         );
 
         $contextualizer->mapConfigArray('ezpage', $config, ContextualizerInterface::MERGE_FROM_SECOND_LEVEL);
-
-        // filters blocks and layouts for each siteaccess to keep only
-        // the enabled ones for this sa
-        foreach ($config['siteaccess']['list'] as $sa) {
-            $ezpageSettings = $container->getParameter("ezsettings.$sa.ezpage");
```

Comment:
I still have a problem at this point. Here, a parameter was loaded without ConfigResolver. Any recommandations to fix this?